server/routes/user.ts

```diff
@@ -213,20 +217,34 @@ router.post('/import-from-plex', async (req, res, next) => {
     const createdUsers: User[] = [];
     for (const rawUser of plexUsersResponse.MediaContainer.User) {
       const account = rawUser.$;
+
       const user = await userRepository.findOne({
-        where: { plexId: account.id },
+        where: [{ plexId: account.id }, { email: account.email }],
       });
+
       if (user) {
         // Update the users avatar with their plex thumbnail (incase it changed)
         user.avatar = account.thumb;
         user.email = account.email;
-        user.username = account.username;
+        user.plexUsername = account.username ?? account.email;
```

Comment:
The account username is always set. We don't have to do the fallback ourselves. Although it doesn't really affect anything here to fall back on our own, it will actually never get called.